cypress/support/commands.ts

```diff
@@ -47,8 +51,7 @@ export function assertNoTab(tabName: string): any {
 export function resetDB() {
   before(() => {
     cy.exec('java -jar dockstore-webservice.jar db drop-all --confirm-delete-everything travisci/web.yml');
-    // cy.exec('docker exec -i postgres1 psql webservice_test -U postgres < travisci/db_dump.sql');
-    cy.exec('PGPASSWORD=dockstore psql -h localhost -f travisci/db_dump.sql webservice_test -U dockstore');
```

Comment:
this change may need more explanation, loading as postgres (the superuser) instead of dockstore can break certain features like frozen versions.

Reply:
changed

Reply:
for this PR, it was already like this. i think i originally put that in for local testing with Docker and didn't really put too much effort in making it correct